README.md

```
@@ -134,6 +134,15 @@ sequenceDiagram
 ## Setup & Deployment
 
 ### Frontend Setup
+
+Setup environment variables:*
+  - add .env file in `/frontend`directory.
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)